classes/apirest/apirest.php

```diff
@@ -318,6 +264,15 @@ public function create_credential_legacy($recipientname, $recipientemail,
         return $this->client->post("{$this->apiendpoint}credentials", $data);
     }
 
+    /**
+     * Get Group
+     * @param int $id
+     * @return stdObject
+     */
+    public function get_group($id) {
```

Comment:
We are not using this method now as we don't need to fetch the group name but I don't mind to keep it as it may be useful in the future.